schemas/+rig_maintenance/RigMaintenanceLog.m

```diff
@@ -0,0 +1,11 @@
+%{
+# Specific actions performed for maintenance on each rig
+-> rig_maintenance.RigMaintenanceAction
+maintenance_time              : DATETIME
+---
+(maintenance_person) ->lab.User	                  
+maintenance_params			  : longtext	    # Json with parameters specific for maintenance action
+%}
+
+classdef RigMaintenanceLog < dj.Manual
```

Comment:
Could we combine RigMaintenanceLog with RigMaintenanceAction?

Reply:
Yes, that is a great idea.
I think it is somewhat redundant RigMaintenanceAction. I will drop that table.